### PR TITLE
Hello

### PR DESCRIPTION
**Pull request by danielschwarz-neuefische**

### CODE REVIEW

File: src/main/java/com/example/cgnjava222gitbranches/HelloController.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController {
+
+    @GetMapping
+    public String sayHello() {
+        return "Hello!";
+    }
```
Comment (christophersiem) [inline src/main/java/com/example/cgnjava222gitbranches/HelloController.java]:
```suggestion
```

File: src/main/java/com/example/cgnjava222gitbranches/HelloController.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    @GetMapping
+    public String sayWelcome() {
+        return "Welcome!";
```
Comment (christophersiem) [inline src/main/java/com/example/cgnjava222gitbranches/HelloController.java:11]: like this endpoint! :)

**Review by christophersiem — CHANGES_REQUESTED** [review]
great work!

File: src/main/java/com/example/cgnjava222gitbranches/HelloController.java (hunk 1 above)
Comment (danielschwarz-neuefische) [reply src/main/java/com/example/cgnjava222gitbranches/HelloController.java]: Danke

**Review by christophersiem — APPROVED** [review]

**Pull request merged**